Dark.json

```diff
@@ -53595,6 +53595,291 @@
         "lore": "Docking bay security officer. Leader. Honored for bravery aboard the Star Destroyer Thunderflare where he rescued five officers from the wreckage of a shuttle crash."
       },
       "legacy": false
+    },
+    {
+      "id": 6850,
+      "side": "Dark",
+      "rarity": "U1",
+      "set": "Virtual Set 14",
+      "front": {
+        "title": "•Ochi",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/Virtual14-Dark/large/ochi.gif?raw=true",
```

Comment:
The image urls for Set 14 don't need "/Images-HT/starwars/" or "?raw=true"
http://res.starwarsccg.org/cards/Virtual14-Dark/large/ochi.gif